next.config.mts

```diff
@@ -1,6 +1,20 @@
 import "./env/server.ts"
 
 /** @type {import('next').NextConfig} */
-const nextConfig = {};
+const nextConfig = {
+  images: {
+    unoptimized: true,
```

Comment:
Doesn't seem to be required.
Ref: https://nextjs.org/docs/app/building-your-application/optimizing/images#remote-images